play/src/pages/GameInProgress.tsx

```diff
@@ -1,32 +1,67 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { makeStyles } from '@mui/styles';
 import { Typography } from "@mui/material";
-import { GameSessionState } from '@righton/networking'
-import { ITeam } from '@righton/networking';
+import { GameSessionState, ModelHelper, ITeam, IQuestion, IChoice, isNullOrUndefined } from '@righton/networking';
+import { v4 as uuidv4 } from 'uuid';
 import HeaderContent from '../components/HeaderContent';
+import CardAnswer from '../components/CardAnswer';
 import FooterContent from '../components/FooterContent';
 
 interface GameInProgressProps {
   teams?: ITeam[];
   id: string;
   currentState: GameSessionState;
   teamAvatar: number;
+  questions: IQuestion[];
+  currentQuestionIndex?: number | null;
+  teamId: string;
 }
 
-export default function GameInProgress( {id, teams, teamAvatar, currentState}: GameInProgressProps) {
+export default function GameInProgress({
+  teams,
+  id,
+  currentState,
+  teamAvatar,
+  questions,
+  currentQuestionIndex,
+  teamId 
+}: GameInProgressProps) {
   const classes = useStyles();
-  const [gameSessionState, setCurrentState] = React.useState(currentState);
-  const currentTeam = !teams ? {name: 'team'} : teams[0];
+  const currentTeam = teams?.find(team => team.id === teamId);
+  const currentQuestion = questions[currentQuestionIndex ?? 0];
+  let teamAnswers;
+  if (currentTeam != null) {
+    teamAnswers = ModelHelper.getBasicTeamMemberAnswersToQuestionId(currentTeam, currentQuestion.id);
+  }
+
+  const [timerIsPaused, setTimerIsPaused] = useState<boolean>(false);
+  const [isSubmitted, setIsSubmitted] = useState<boolean>(false);
+  const [selectedAnswer, setSelectedAnswer] = useState<number | null>(null);
+
+  const answerChoices = currentQuestion?.choices?.map((choice: IChoice) => {
+    return {
+        id: uuidv4(),
+        text: choice.text,
+        isCorrectAnswer: choice.isAnswer,
+    }
+  })
 
   const handleTimerIsFinished = () => {
-    console.log('finished');
+    setTimerIsPaused(true);
```

Comment:
Just a note, have tested this when user switches tabs as well?

Reply:
good point, the timer currently doesn't pause for anything. I'll need to add that in but I'd like to stay focused on broader UI and then fine tune that functionality in a subsequent pull request. To make sure I'm not just kicking the can down the road, I've added a TODO section to the readme to chart these items (with links to new Github issues)